onthego/spotify.py

```diff
@@ -62,7 +65,7 @@ def iter_playlists(self):
 
 class Track(object):
 
-    def __init__(self, name, artists, album):
+    def __init__(self, name, artists, album={'name': ''}):
```

Comment:
I can see two problems with this solution:
1. it's dangerous to pass mutable objects as default keyword arguments (http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments)
2. the default argument defined here will cause the `album_art_url` and `album_release_date` properties to crash with a `KeyError`.